library/clefos

```diff
@@ -2,7 +2,7 @@ Maintainers: The ClefOS Project <neale@sinenomine.net> (@nealef)
 GitRepo: https://github.com/nealef/clefos.git
 Constraints: !aufs
 
-Tags: 7, 7.5.1804, latest
-GitFetch: refs/heads/7.4.1708
-GitCommit: a13384382290c7aef7012d1353fee4f84ce8548c
+Tags: 7, 7.6.1810, latest
+GitFetch: refs/heads/7.6.1810
```

Comment:
This new commit appears to be on the `7.4.1708` branch still (which is totally fine -- in fact, I'd recommend renaming it something like `dist` and just re-using it every time).

Reply:
Would you like me to make that change now or wait until the next time?

Reply:
You definitely need to either do that change or adjust this file to point at the old branch name (which would be harmless otherwise).  Whichever is easier for you.  If you'd like, I'd be happy to update this PR for you and get it merged in so you can worry about the repo organization for next time instead?

Reply:
Yes please! Thanks again.